frontend/src/app/render_wasm/shape.cljs

```diff
@@ -19,6 +19,15 @@
 (declare ^:private impl-assoc)
 (declare ^:private impl-conj)
 (declare ^:private impl-dissoc)
+(defonce ^:private current-page-object-ids #{})
+
+(defn set-current-page-object-ids!
+  [objects]
+  (set! current-page-object-ids (set (keys objects))))
```

Comment:
This operation has more overhead than just having the whole map. So I suggest to put all the objects there and
rename the function to `set-current-page-objects!`

Reply:
ok! let's do it that way then, I'll change that back